content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -85,7 +85,7 @@ different Kubernetes components.
 | `CustomCPUCFSQuotaPeriod` | `false` | Alpha | 1.12 | |
 | `CustomResourceDefaulting` | `false` | Alpha| 1.15 | 1.15 |
 | `CustomResourceDefaulting` | `true` | Beta | 1.16 | |
-| `DefaultPodTopologySpread` | `false` | Alpha | 1.19 | |
+| `DefaultPodTopologySpread` | `true` | Beta | 1.20 | |
```

Comment:
I believe we don't remove it and just add on to it to show the history of the feature gate

```suggestion
| `DefaultPodTopologySpread` | `false` | Alpha | 1.19 | 1.19 |
| `DefaultPodTopologySpread` | `true` | Beta | 1.20 | |
```

Reply:
Done

Reply:
There is one more change to make. Please move these rows to be under the heading “Feature gates for graduated or deprecated features”.

Reply:
This is beta

Reply:
D'oh! Many apologies.